forage-android/src/main/java/com/joinforage/forage/android/core/ui/element/state/pin/PinEditTextState.kt

```diff
@@ -6,7 +6,7 @@ import com.joinforage.forage.android.core.ui.element.state.FocusState
 
 /**
  * An interface that represents the state of a
- * [ForagePINEditText][com.joinforage.forage.android.core.ui.ForagePINEditText] Element.
+ * **ForagePINEditText** Element.
```

Comment:
can't use a Kdoc link here, because it's in `core` but `ForagePINEditText` is in `ecom` 😢 , so i bolded instead!